src/test/java/com/camping/admin/steps/ReservationSteps.java

```diff
@@ -0,0 +1,48 @@
+package com.camping.admin.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReservationSteps {
+    private int reservationId;
+
+    @When("관리자가 예약 상태를 \"CONFIRMED\"로 변경한다.")
+    public void 관리자가예약상태를CONFIRMED로변경한다() {
```

Comment:
이렇게 인자를 받게 처리 할 수 도 있겠네요 :) 

```suggestion
    @When("관리자가 예약 상태를 {string} 로 변경한다.")
    public void 관리자가예약상태를변경한다(String string) {
```